docs/team/shummyownzyou.md

```diff
@@ -5,30 +5,40 @@ title: ShummyOwnzYou's Project Portfolio Page
 ### Project: TrackAScholar
 
 
-Given below are my contributions to the project.
+TrackAScholar is a desktop application used for managing scholarship applications.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX.
+It is written in Java, and has about 14 kLoC.
 
-* **New Feature**: Added `FilterCommand` and `ApplicationStatusPredicate` class under `applicant` package with Benjy
+* **New Feature**: Added the ability to filter applicants by application status together with Benjy: [#56](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/56), [#62](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/62), [#68](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/68)
```

Comment:
You can credit benjy under the justification.

* Credits: Developed in tandem with Benjy's help

or something similar